vault/external_tests/approle/wrapped_secretid_test.go

```diff
@@ -76,31 +52,11 @@ func TestApproleSecretId_Wrapped(t *testing.T) {
 }
 
 func TestApproleSecretId_NotWrapped(t *testing.T) {
-	var err error
-	coreConfig := &vault.CoreConfig{
-		DisableMlock: true,
-		DisableCache: true,
```

Comment:
Would these change anything for this test? It seems they are not set in the minimal one.

Reply:
Not that I'm aware.  I think the author was copy-pasting blindly.